packages/api/src/controllers/session.ts

```diff
@@ -145,13 +145,24 @@ app.get("/:id", authorizer({}), async (req, res) => {
   let session = await db.session.get(req.params.id);
   if (
     !session ||
-    ((session.userId !== req.user.id || session.deleted) && !req.user.admin)
+    ((session.userId !== req.user.id || session.deleted) &&
+      !req.user.admin &&
+      !LVPR_SDK_EMAILS.includes(req.user.email))
   ) {
     // do not reveal that session exists
     res.status(404);
     return res.json({ errors: ["not found"] });
   }
   res.status(200);
+
+  let originalRecordingUrl: string | null = null;
+
+  if (req.query.sourceRecording == "true") {
+    const { url } = await getSourceRecordingBySessionId(req.params.id, req);
+    originalRecordingUrl = url;
+    session.sourceRecordingUrl = originalRecordingUrl;
```

Comment:
Keep in mind that these will not be available forever right. Do we have any check to return a clear error to the user when the recording is already deleted?

Reply:
@emranemran How long are these source recording saved?

Reply:
6 days